Async-development/script.js

```diff
@@ -0,0 +1,122 @@
+
+
+document.querySelector('#form').addEventListener('submit', showWeatherBlock);
+
+function showWeatherBlock(event) {
+    event.preventDefault();
+    let city = document.querySelector('#city').value;
+    let country = document.querySelector('#country').value;
+    let promise = new Promise((resolve, reject) => {
+        let response= fetch(`http://api.weatherstack.com/current?access_key=398b72a14594b34c8ae70e20cc025c73&query=${city},${country}`);
+        response.then(result => {return result.json()}).
+        then(data => {
+            if (data.success===false) {
+                reject(createErrorBlock());
+                
+            } else {
+                console.log(data);
+                resolve(data);
+            }
+        })
+      });

```

Comment:
no need to wrap with `new Promise`
```suggestion
    let promise = fetch(`http://api.weatherstack.com/current?access_key=398b72a14594b34c8ae70e20cc025c73&query=${city},${country}`)
    .then(result => {return result.json()}).
        then(data => {
            if (data.success===false) {
                createErrorBlock();
            } else {
                createWeatherBlock(result)
            }
        });

```